src/index.html

```diff
@@ -5,8 +5,314 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="MoyoLogo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list__item">
+            <a href="#apple" class="list__link is-active">Apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#samsung" class="list__link">Samsung</a>
+          </li>
+          <li class="list__item">
+            <a href="#smartphones" class="list__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#pc"
+              class="list__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="list__item">
+            <a href="#gadgets" class="list__link">Гаджеты</a>
+          </li>
+          <li class="list__item">
+            <a href="#tablets" class="list__link">Планшеты</a>
+          </li>
+          <li class="list__item">
+            <a href="#photo" class="list__link">Фото</a>
+          </li>
+          <li class="list__item">
+            <a href="#video" class="list__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <section class="card" data-qa="card">
```

Comment:
Better add class `page` to body and set other blocks to it`s element. This way easer write code for a project without unnecessary nesting.

 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

This time you could leave code that you have(because you have no container in `header` but for next projects with two or more blocks(header, catalog, testimonials...) use `mix` and `page` class for body(watch video).
```suggestion
    <main>
      <div class="page__catalog catalog">
        <section class="card" data-qa="card">
```